CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [3.0.9](https://github.com/ng-select/ng-select/compare/v3.0.8...v3.0.9) (2019-09-02)
```

Comment:
This change is not needed. Changelog is updated when doing a new release.

Reply:
@anjmao my fault. sorry